ironfish/src/assert.ts

```diff
@@ -91,4 +91,13 @@ export class Assert {
       throw new Error(`We must have forgotten a falsey value: ${String(x)}`)
     }
   }
+
+  static bufferEquals(x: Buffer, y: Buffer, message?: string): void {
```

Comment:
I looked around if a similar function already exists but couldn't find any. I wouldn't be surprised though if there's already something similar